kong/core/router.lua

```diff
@@ -446,25 +446,24 @@ function _M.new(apis)
   end)
 
 
-  local grab_headers = #wildcard_hosts > 0 or next(indexes.plain_hosts) ~= nil
+  local grab_host = #wildcard_hosts > 0 or next(indexes.plain_hosts) ~= nil
```

Comment:
might we consider removing this variable altogether and moving this expression to line 582? reducing the number of live local variables is a win (this assumes the commented out print statement on 579 is removed or adjusted as well)